client/src/Molecules/Footer/FooterDesktop.js

```diff
@@ -0,0 +1,86 @@
+import FooterDesktopInnerStyled from './FooterDesktopInnerStyled';
```

Comment:
I think this file might be a duplicate? I cant find where you are using it?

Reply:
You are correct. Deleted.